static/app/views/relocation/getStarted.tsx

```diff
@@ -80,7 +80,6 @@ function GetStarted({relocationState, onUpdateRelocationState, onComplete}: Step
             onChange={evt => {
               onUpdateRelocationState({orgSlugs: evt.target.value});
             }}
-            required
```

Comment:
cc @hubertdeng123 this never worked due to https://github.com/getsentry/sentry/blob/a3096e5c1dafb4a2afec2fbe62a5dfae7a1881df/static/app/components/core/input/index.tsx#L91